src/client.rs

```diff
@@ -19,6 +19,7 @@ static API_URL: &str = "https://api.sendgrid.com/api/mail.send.json?";
 #[derive(Clone, Debug)]
 pub struct SGClient {
     api_key: String,
+    host: Option<String>,
```

Comment:
Maybe we can just have this be a `String` and set it to the `API_URL` value in `new`? IMO we should always have a `host`.

The `set_host` function can just mutate the value.

Reply:
Great point, will make that change. Would it be better for backwards compatibility to create a new constructor such as `new_with_host` so that existing calls to `new` continue to work?